openmdao/visualization/meta_model_viewer/tests/test_view_mm_cmd_line.py

```diff
@@ -16,20 +16,30 @@ def test_unspecified_metamodel(self):
         script = os.path.join(os.path.dirname(__file__), 'example.py')
         cmd = 'openmdao view_mm {}'.format(script)
         output = subprocess.check_output(cmd.split()).decode('utf-8', 'ignore')
-        expected_output = ("Metamodel not specified. "
-                           "Try one of the following: ['interp1', 'interp2'].")
+        expected_output = ('\nMetamodel not specified. Try the following:\n'
+                           '\nopenmdao view_mm -m interp1 {}'
+                           '\nopenmdao view_mm -m interp2 {}\n'.format(script, script))
         self.assertTrue(
-            expected_output in output,
-            msg='Metamodel was specified when it should not have been. Check example.')
+            expected_output in output)
 
     def test_invalid_metamodel(self):
-        script = os.path.abspath(example.__file__).replace('.pyc', '.py') # PY2
+        script = os.path.abspath(example.__file__)
         cmd = 'openmdao view_mm {} -m {}'.format(script, 'interp')
         output = subprocess.check_output(cmd.split()).decode('utf-8', 'ignore')
-        expected_output = '\n'.join([
-            "Metamodel 'interp' not found.",
-            " Try one of the following: ['mm']."
-        ])
+        expected_output = (
+            "\nMetamodel 'interp' not found. Try one of the following:\n"
+            "\nopenmdao view_mm -m mm {}\n".format(script)
```

Comment:
I still think you could improve this by only using the "one of" text if there are multiple mm's...
This example has a single MM and says "one of" while the example in L19 with multiple options does not.

also these tests are currently failing on Windows...